### DISCUSSION TITLE
YouTube Streaming Issue

### DISCUSSION THREAD

**Discussion by drankiebeverly** [ant-media/Ant-Media-Server#5181] [Q&A]
Hello, 

I am having issues with streaming using your servers. The video in question does have audio and there is no space in the RTMP endpoint. It also seems as though the video isn't actually playing when I preview it on the live stream but when I check the VOD section the video plays just fine.  I am using the community version and I'm not sure if that is the reason why. But if so I would like to know why or at the very least how to fix this. Thank you in advance for your help. 

![Screenshot 2023-06-01 173003](https://github.com/ant-media/Ant-Media-Server/assets/135279630/2f9232f0-f9b7-4f33-a14a-f75428ba6465)

**Comment by yashtandon113** [comment]
@drankiebeverly 
I hope you are doing well.
Can you please share the video that you are using to stream? I will check on my end.

**Reply by mekya** [reply]
Just let me add something to this. 

There was a [youtube video about](https://www.youtube.com/watch?v=NVhYthQk_js&ab_channel=AntMediaServer) this. Maybe you've already watched, then I'm just putting this for future users 

Secondly, as far as I remember one platform(youtube or facebook) doesn't accept live streams if there is no audio. 

Lastly, locating the problem is important. 
Just use your camera and microphone(without using video file) and check that if it works. 
If it works, then it means the server side is working and there should be something in the video file. 
If it does not work, it's an issue that we need to investigate and understand why it's not working.

**Reply by USAMAWIZARD** [reply]
I checked the VOD file on the tickets. He is using H265 encoded VOD file which is not supported over rtmp youtube https://stackoverflow.com/questions/61996884/does-youtube-support-live-ingest-of-hevc-over-rtmp-or-is-this-only-available-via.

**Comment by USAMAWIZARD** [comment]
The reason that you are not getting the video on YouTube side is  because  RTMP does not support H265 encoded Video for stream to YouTube.
You can convert the video to h264 by using various tools available online.
ffmpeg -i in.mp4 -c:v libx264 -crf 18 -vf format=yuv420p -c:a copy out.mp4
